lib/api/controllers/collectionController.js

```diff
@@ -341,6 +319,31 @@ class CollectionController extends BaseController {
         return response;
       });
   }
+
+  /**
+   * @param {Request} request
+   * @returns {Promise.<Array>}
+   */
+  _createSpecificationList (index, collection, specifications) {
+    if (index && collection) {
+      const specification = formatSpecification(
+        index,
+        collection,
+        specifications);
+      return Bluebird.resolve([ specification ]);
+    }
+
+    const specifications = [];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "specifications" as this name is already used in declaration at line 327. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS2814)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "specifications". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1226)